cmake/serac-config.cmake.in

```diff
@@ -47,6 +47,7 @@ if(NOT SERAC_FOUND)
   set(SERAC_USE_SUNDIALS       @SERAC_USE_SUNDIALS@)
   set(SERAC_USE_TRIBOL         @SERAC_USE_TRIBOL@)
   set(SERAC_USE_UMPIRE         @SERAC_USE_UMPIRE@)
+  set(SERAC_USE_ENZYME         @SERAC_USE_ENZYME@)
```

Comment:
I took out the `SERAC_USE_ENZYME` stuff in the Enzyme PR, so we probably want to get rid of it here.  I don't think Serac will have anything enzyme'd in it directly, just through mfem and Tribol.

Reply:
I forgot about the different tolerances in the test if we're using enzyme; let's leave this in...